art/estimators/object_detection/PyTorchFasterRCNN.py

```diff
@@ -51,7 +51,7 @@ def __init__(
         channels_first: Optional[bool] = None,
         preprocessing_defences: Union["Preprocessor", List["Preprocessor"], None] = None,
         postprocessing_defences: Union["Postprocessor", List["Postprocessor"], None] = None,
-        preprocessing: "PREPROCESSING_TYPE" = (0, 1),
+        preprocessing: Optional["PREPROCESSING_TYPE"] = None,
```

Comment:
I think adding the `None` here is ok, but because it's valid for all estimators I would like to move the `Optional` to the definition of `PREPROCESSING_TYPE` in `art.config` in line 35 to make that line
```PREPROCESSING_TYPE = Optional[Tuple[Union[int, float, np.ndarray], Union[int, float, np.ndarray]]]```